sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue

```diff
@@ -1,103 +1,122 @@
 <template>
-  <modal-dialog-base
-    title="Application already in progress"
-    dialogType="warning"
-    :showDialog="showDialog"
-    @dialogClosed="dialogClosed"
-  >
-    <template v-slot:content>
-      <v-container>
-        <form>
-          <p>There is already a draft of an application in progress.</p>
-          <p>Please continue your draft application or cancel it.</p>
-        </form>
-      </v-container>
-    </template>
-    <template v-slot:footer>
-      <footer-buttons
-        secondaryLabel="Close"
-        :showPrimaryButton="false"
-        @secondaryClick="dialogClosed"
-      />
+  <student-page-container :layout-template="LayoutTemplates.Centered">
+    <template #header>
+      <header-navigator
+        title="Applications"
+        subTitle="Start New Application"
+        :routeLocation="{
+          name: StudentRoutesConst.STUDENT_APPLICATION_SUMMARY,
+        }"
+      >
+      </header-navigator>
     </template>
-  </modal-dialog-base>
-  <student-page-container>
-    <v-sheet elevation="1" class="mx-auto">
-      <v-container>
-        <formio
-          formName="studentapplication"
-          :data="initialData"
-          @loaded="formLoaded"
-          @customEvent="customEventCallback"
-        ></formio>
-      </v-container>
-    </v-sheet>
+    <v-form ref="startApplicationForm">
+      <h2 class="category-header-large primary-color pb-4">
+        Apply for new funding
+      </h2>
+      <content-group>
+        <h3 class="category-header-medium-small pa-2">
+          Financial Aid Application
+        </h3>
+        <p class="px-2">
+          Apply to see if you are applicable for provincial or federal loans and
```

Comment:
looks like the content is different in Figma, 
![image](https://user-images.githubusercontent.com/77353155/199856232-329202ba-c01e-44b3-a33f-082a701e2571.png)
'
or am i missing anything

Reply:
Actually this screen is in two parts and they are different:
![image](https://user-images.githubusercontent.com/78114138/199856723-220c9785-eac0-4d51-9bcb-b4a23509eff5.png)

@hellolynn-tbtb could you confirm which is one is correct?

Reply:
@andrepestana-aot hey andre you have it correct! I fixed it now in Figma. My apologies - thanks for both of your eyes! @ann-aot

Reply:
@ann-aot, Lynn fixed that on figma and I changed accordingly. Thanks!